gradle-plugin/src/main/kotlin/com/google/devtools/ksp/gradle/KspAATask.kt

```diff
@@ -128,6 +136,19 @@ abstract class KspAATask @Inject constructor(
                     }
                     cfg.logLevel.value(logLevel)
                     cfg.allWarningsAsErrors.value(kspExtension.allWarningsAsErrors)
+                    cfg.excludedProcessors.value(kspExtension.excludedProcessors)
+
+                    val jvmDefaultMode = project.provider {
+                        compilerOptions.freeCompilerArgs.get().lastOrNull {
```

Comment:
is `lastOrnull` a standard solution for duplicate arguments?

Reply:
It's a common practice for argument parsing. The latter overrides the former.